.github/workflows/cross-version-tests.yml

```diff
@@ -55,7 +55,7 @@ jobs:
             return labels.data.some(({ name }) => name === "enable-dev-tests");
       - name: Test set_matrix.py
         run: |
-          pytest dev/set_matrix.py --doctest-modules
+          python -m pytest --noconftest tests/dev/test_set_matrix.py --doctest-modules dev/set_matrix.py
```

Comment:
- `--noconftest` to skip loading modules in `conftest.py`.
- `python -m` to include the current directory to `sys.path`.